### ISSUE TITLE
Computing the best scores

### ISSUE THREAD

**Issue by PhyloStar** [lingpy/bipskip#3]
@LinguList I come to understand that the for loop in test.py computes the best bcubed score across the datasets for a setting. The following line is doing an average of the F-scores. I don't think an average is required here.

https://github.com/lingpy/fast-cognate-detection-paper/blob/3322107ecc05110c5a57345820079aee9d30a583/test.py#L41

The bcubes calcualation on https://github.com/lingpy/fast-cognate-detection-paper/blob/3322107ecc05110c5a57345820079aee9d30a583/test.py#L37 is for the whole word list right?

**Comment by PhyloStar** [comment]
How to prevent the following line from coming up when executing the program?
Column <autocog> already exists, do you want to override? [y/N] n

**Comment by LinguList** [comment]
Yes. But the average is across the datasets, right? So it should be computed for all datasets.

**Comment by LinguList** [comment]
As to the line: I think, you have an old lingpy version, as this does not occur in mine. Maybe quickly try to make sure you have 2.6.4 ?

**Comment by PhyloStar** [comment]
Okay. You want to find the best setting across the datasets or across the n-gram setting?

**Comment by LinguList** [comment]
Yes, this is the idea, the "best_fs" is actually not needed, just making it easier to follow the updates, as this test is running quite fast...

**Comment by PhyloStar** [comment]
> As to the line: I think, you have an old lingpy version, as this does not occur in mine. Maybe quickly try to make sure you have 2.6.4 ?

I am using Lingpy 2.6.4.alpha

**Comment by LinguList** [comment]
Okay, you need to update, best to git-clone lingpy-2.6.4. Or just the one on github. in 2.6.4.alpha, we still had a problem with "override".

**Comment by PhyloStar** [comment]
> Okay, you need to update, best to git-clone lingpy-2.6.4. Or just the one on github. in 2.6.4.alpha, we still had a problem with "override".

I installed it from git.

**Comment by LinguList** [comment]
then let me know in which line among output it asks...

**Comment by LinguList** [comment]
And update your git: the 2.6.4.alpha is not the most recent version, the most recent one is 2.6.4...

**Comment by LinguList** [comment]
sorry, another test is: use the "override" keyword. If you cluster cognates, write: `override=True` inside the cluster function. This prevents from asking, if you override internal cogids.

**Comment by PhyloStar** [comment]
Okay. I was able to run without autocog line showing up.
